GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -25,6 +25,12 @@ public class StoreTests : DatabaseTests
         private readonly Mock<IGeocodeClientAdapter> _mockGeocodeClient;

```

Comment:
I think we need to split the teaching event building sync specs out of the teaching event sync specs now as the behaviour has changed (and we sync buildings without events whereas the existing tests only cover buildings with events), so have:

```
public async void SyncAsync_InsertsNewTeachingEventBuildings()
public async void SyncAsync_UpdatesExistingTeachingEvents()
public async void SyncAsync_DeletesOrphanedTeachingEventBuildings()
```